uint32 seq
int32 voice_id
int32 person_id
int32 pi_id
int32 group_id
int32[] people_in_group
string text